test/run-windows-k8s-integration.sh

```diff
@@ -23,7 +23,7 @@ base_cmd="${PKGDIR}/bin/k8s-integration-test \
             --platform=windows --bringup-cluster=false --teardown-cluster=false --teardown-driver=${teardown_driver}\
             --run-in-prow=true --deploy-overlay-name=${overlay_name} --service-account-file=${E2E_GOOGLE_APPLICATION_CREDENTIALS} \
             --do-driver-build=${do_driver_build} --gce-zone=${gce_zone} --test-version=${test_version}\
-            --storageclass-files=sc-windows.yaml --snapshotclass-file=pd-volumesnapshotclass.yaml --test-focus='External.Storage' \
+            --storageclass-files=sc-windows.yaml --snapshotclass-file=pd-volumesnapshotclass.yaml --test-focus='External.Storage.*ntfs.*[sig-windows]' \
```

Comment:
We need [sig-windows] because some test cases in the external storage framework are known to not work on Windows? Would it be better to have a skip on [LinuxOnly] instead?

Reply:
I think it is ok to remove sig-windows. updated